src/Create.js

```diff
@@ -0,0 +1,31 @@
+import React, {useState, useEffect} from 'react';
+import Firestore from './services/firestore.js';
+import Board from './Board';
+
+function Create(props) {
+  const [grid, setGrid] = useState([]);
+
+  async function saveGrid(e) {
+    e.preventDefault();
+    let fields = e.target.elements;
+    let email = fields['email'].value;
+    let title = fields['title'].value;
+    const createdGrid = await Firestore.add.grid(email, title, grid);
+    if (createdGrid === null) { /* error */ }
```

Comment:
Maybe displaying an error message to the user would be helpful

Reply:
Done.